tests/unit_tests/bittensor_tests/test_forward_backward.py

```diff
@@ -11,6 +12,7 @@
 dendrite.receptor_pool.forward = MagicMock(return_value = [torch.tensor([]), [0]]) 
 dendrite.receptor_pool.backward = MagicMock(return_value = [torch.tensor([]), [0]]) 
 endpoint = bittensor.endpoint(
+    version = bittensor.__version_as_int__,
```

Comment:
Ah I see what you did. You added version_as_int in another PR. either way, let's make the int calculation more automated, manual calculation is going to lead to problems later. See #498 for my suggestion on it.